lib/hammer_cli_foreman/commands.rb

```diff
@@ -740,17 +740,18 @@ def request_options
     option "--path", "PATH", _("Path to directory where downloaded content will be saved"),
         :attribute_name => :option_path
 
-    def execute
+    def execute(source = "response")
```

Comment:
I wouldn't change the `execute` method's signature just for this. What prevents us from creating a new public/protected method, which will just return a string? We could then easily override that method in descendants .

Reply:
didn't think about it :D 
Thanks! I'll fix it